keyloop/schemas/path.py

```diff
@@ -18,8 +19,9 @@ def validate_realm(self, value):
         registry = request.registry.settings["keyloop_adapters"]
 
         identity_provider = registry.lookup([IIdentity], IIdentitySource, value)
+        auth_session = registry.lookup([IAuthSession], IAuthSessionSource, value)
 
-        if not identity_provider:
+        if not identity_provider and not auth_session:
```

Comment:
Acho que ficaria mais legível dessa forma:
```suggestion
        if not (identity_provider or auth_session):
```